app/src/main/java/gg/strims/android/fragments/StreamsFragment.kt

```diff
@@ -49,7 +48,13 @@ class StreamsFragment : Fragment() {
                 it.live
             }
             CurrentUser.streams!!.forEach {
-                streamsAdapter.add(StreamItem(it))
+                if (!CurrentUser.user!!.show_hidden) {
+                    if (!it.hidden) {
+                        streamsAdapter.add(StreamItem(it))
+                    }
```

Comment:
can we not merge this if statement into 1 line idk kotlin `if (!CurrentUser.user!!.show_hidden && !it.hidden)`

Reply:
Changes made.